cognite/client/data_classes/data_modeling/instances.py

```diff
@@ -389,7 +401,16 @@ def __contains__(self, attr: str) -> bool:
             self.__raise_if_non_singular_source(attr)
 
     def dump(self, camel_case: bool = True) -> dict[str, Any]:
-        dumped = super().dump(camel_case)
```

Comment:
We cannot call the default dump method, as the TypedNode/Edge, will have properties directly on the class, which the default method just serialize in place.